assets/js/main.js

```diff
@@ -23,11 +23,11 @@ function serialize(a,b){"object"!=typeof b?b={hash:!!b}:void 0===b.hash&&(b.hash
 
 /**
  * jQuery.query - Query String Modification and Creation for jQuery
- * Written by Blair Mitchelmore (blair DOT mitchelmore AT gmail DOT com)
+ * Written by Blair MitchcookieModalre (blair DOT mitchcookieModalre AT gmail DOT com)
  * Licensed under the WTFPL (http://sam.zoy.org/wtfpl/).
  * Date: 2009/8/13
  *
- * @author Blair Mitchelmore
+ * @author Blair MitchcookieModalre
```

Comment:
Looks like accidental search/replace?

Reply:
Yea, I was replacing the name of cookie modal "elmo" to just "cookieModal". I'll fix it.